src/Components/Tile.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+
+
+function Tile({tileImage, tileHeader, extraParagraph}){
+
+    const placeholder = "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur atque deleniti fugiat hic id, necessitatibus nostrum vel. Aliquid blanditiis cum dicta dolore illo ipsam magni necessitatibus praesentium quos ut? Velit."
+
+    if (tileImage !== undefined){
```

Comment:
Slim opgelost! En we hadden {Children} ook nog niet in de les gehad. Maar het kan dus veel korter:

```
function Tile({ img, imgDescription, title, children }) {
  return (
    <section>
      {img && <img src={img} alt={imgDescription}/>}
      <h2>{title}</h2>
      {children}
    </section>
  );
}

```